tests/test_ipv6.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -e
+stdbuf -o0 ./twamp-light-server -n 10 --ip 6 -P 4200 &> server_output.txt &
```

Comment:
what happens whenn calling a ipv6 server url with -4 and or a ipv4 server url with -6?